packages/AlgernonTests.package/ALGTestCache.class/instance/testCacheKeyGenerationLevel1.st

```diff
@@ -1,7 +1,5 @@
 testing
 testCacheKeyGenerationLevel1
-
-	| parent |
-	parent := ALGItem new.
+	ALGItem new.
```

Comment:
I am not sure whether this is necessary for the test.

Reply:
It should not be, a similar thing got removed here: https://github.com/hpi-swa-teaching/Algernon-Launcher/pull/155/files#diff-b1bbfaab62f80e23ef1bc09277ff9fd5